core/src/main/java/com/linecorp/armeria/common/MediaType.java

```diff
@@ -1052,6 +1054,20 @@ public boolean isJson() {
         return is(JSON) || subtype().endsWith("+json");
     }
 
+    /**
+     * Returns {@code true} when the subtype is in [{@link MediaType#PROTOBUF}, {@link MediaType#X_PROTOBUF}, {@link MediaType#X_GOOGLE_PROTOBUF}].
```

Comment:
We prefer to list elements using `and` or `or`.
```suggestion
     * Returns {@code true} when the subtype is one of {@link MediaType#PROTOBUF}, {@link MediaType#X_PROTOBUF}, and {@link MediaType#X_GOOGLE_PROTOBUF}.
```